json/src/de/mod.rs

```diff
@@ -535,4 +535,41 @@ mod tests {
 
         assert!(super::from_value::<InMemDicomObject>(serialized).is_ok());
     }
+
+    #[test]
+    fn can_resolve_nan_and_inf_float() {
+        let serialized = serde_json::json!({
+            "00020011": {
```

Comment:
A slightly more realistic example would probably use a tag for FL values, such as Table of Parameter Values (0018,605A).

Reply:
Updated